content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Getting started with Integrations
+kind: documentation
+aliases:
+  - /getting-started-integrations/
+  - /guides/getting-started-integrations/
+  - /guides/integrations/
+further_reading:
+- link: "https://learn.datadoghq.com/course/view.php?id=2"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Our full list of integrations"
+- link: "/graphing/"
+  tag: "Graphing"
+  text: "Now that you have your integrations set up, build graphs to visulize your data"
+- link: "/monitors/"
+  tag: "Alerts"
+  text: "Set up alerts on your data"
+---
+
+An integration at the highest level is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole - you can see pieces individually and also how individual pieces are impacting the whole. Datadogs integrations use a Python class called a check to define the typical metrics most people would want to monitor for that system.
```

Comment:
```suggestion
An integration, at the highest level, is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole—you can see pieces individually and also how individual pieces are impacting the whole. Datadog's integrations use a Python class called `check` to define the typical metrics most people would want to monitor for that system.
```

I'm not sure now necessary it is to explain integrations at such a high level. I think even our least technical users are still a bit technical enough to know what "integration" means in this context. It is kind of nice to have the explanation there, though... Things to think about.

Reply:
Yeah, this is an interesting point to think about. I didn't spend too much time on it, but Kirk told me that this is actually a question that comes up a lot at events.